src/CSharp/EasyMicroservices.Serialization.BinaryGo/Providers/BinaryGoProvider.cs

```diff
@@ -14,21 +14,21 @@ public class BinaryGoProvider : BaseBinarySerializationProvider
         /// Deserialize from byte
         /// </summary>
         /// <typeparam name="T"></typeparam>
-        /// <param name="reader"></param>
+        /// <param name="value"></param>
         /// <returns></returns>
-        public override T Deserialize<T>(ReadOnlySpan<byte> reader)
+        public override T Deserialize<T>(byte[] value)
```

Comment:
Please don't change the engine structure.
you can change only the message pack provider to return as ReadOnlySpan